src/app/views/collections/details/CollectionDetailsController.jsx

```diff
@@ -176,6 +176,7 @@ export class CollectionDetailsController extends Component {
                     return;
                 }
                 const mappedCollection = collectionMapper.collectionResponseToState(collection, this.props.groups);
+                // Have some stuff going on here?
                 const collectionWithPages = collectionMapper.pagesToCollectionState(mappedCollection);
 
```

Comment:
I think this may be a potential place to put code making requests to /users however here there would need to be some array manipulation and just isn't as easy as it is in details. Operation would occur after mapping but before being passed to details for render